docs/topics/link-extractors.rst

```diff
@@ -10,13 +10,21 @@ The ``__init__`` method of
 :class:`~scrapy.linkextractors.lxmlhtml.LxmlLinkExtractor` takes settings that
 determine which links may be extracted. :class:`LxmlLinkExtractor.extract_links
 <scrapy.linkextractors.lxmlhtml.LxmlLinkExtractor.extract_links>` returns a
-list of matching :class:`scrapy.link.Link` objects from a
+list of matching :class:`~scrapy.link.Link` objects from a
 :class:`~scrapy.http.Response` object.
 
 Link extractors are used in :class:`~scrapy.spiders.CrawlSpider` spiders
 through a set of :class:`~scrapy.spiders.Rule` objects. You can also use link
 extractors in regular spiders.
 
+In other to use link extractors in regular spiders, you should instantiate ``scrapy.linkextractors.LinkExtractor`` into
```

Comment:
Two comments:
* "In other" -> "In order"
* "should" is not completely accurate here, storing link extractors as class/instance variables is not mandatory

Reply:
Thanks for catching those. Will fix.